File: darwin/Classes/AmplitudeFlutterPlugin.m
#import "AmplitudeFlutterPlugin.h"
#import <amplitude_flutter/amplitude_flutter-Swift.h>

@implementation AmplitudeFlutterPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftAmplitudeFlutterPlugin registerWithRegistrar:registrar];
}
@end

File: darwin/amplitude_flutter/Sources/amplitude_flutter/AmplitudeFlutterPlugin.swift
#if os(iOS)
import Flutter
#elseif os(macOS)
import FlutterMacOS
#endif

@objc public class AmplitudeFlutterPlugin: NSObject, FlutterPlugin {
    public static func register(with registrar: FlutterPluginRegistrar) {
        SwiftAmplitudeFlutterPlugin.register(with: registrar)
    }
}
